lms/djangoapps/bulk_email/models.py

```diff
@@ -471,6 +471,29 @@ def __str__(self):
         return f"Course '{str(self.course_id)}': Instructor Email {not_en}Enabled"
 
 
+class DisabledCourse(models.Model):
+    """
+    Disable the bulk email feature for specific courses.
+
+    .. no_pii:
+    """
+    class Meta:
+        app_label = "bulk_email"
+
+    course_id = CourseKeyField(max_length=255, db_index=True, unique=True)
+
+    @classmethod
+    def instructor_email_disabled_for_course(cls, course_id):
+        """
+        Returns whether or not email is disabled for the given course id.
+        """
+        try:
+            record = cls.objects.get(course_id=course_id)
```

Comment:
nit: I _think_ this can be simplified ever-so-slightly to something like...
```
return cls.objects.filter(course_id=course_id).exists()
```

If the QuerySet has a result it should return True, else, if the QuerySet is empty, it returns False. That way I believe we can ditch the try/except. I don't think we _need_ the `get` call here because we don't actually want to return the DisabledEmail entry itself, we jut want to check if an entry exists for the given `course_id`.

Reply:
good callout! works just fine, thanks!